src/SFA.DAS.ApplyService.Application.UnitTests/Handlers/AssignAssessorHandlerTests/AssignAssessorHandlerTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Apply;
+using SFA.DAS.ApplyService.Application.Apply.Assessor;
+
+namespace SFA.DAS.ApplyService.Application.UnitTests.Handlers.AssignAssessorHandlerTests
+{
+    [TestFixture]
+    public class AssignAssessorHandlerTests
+    {
+        private Mock<IApplyRepository> _repository;
+        private AssignAssessorHandler _handler;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            _repository = new Mock<IApplyRepository>();
+            _handler = new AssignAssessorHandler(_repository.Object, Mock.Of<ILogger<AssignAssessorHandler>>());
+        }
+
+        [Test]
+        public async Task Assigning_application_to_assessor_1_stored_the_assessor_details()
+        {
+            var applicationId = Guid.NewGuid();
+            var expectedUserId = "sadjkffgdji";
+            var expectedUserName = "sadjkffgdji";
+            
+            await _handler.Handle(new AssignAssessorRequest(applicationId, 1, expectedUserId, expectedUserName), new CancellationToken());
```

Comment:
Using named params would make it easier to see at a glance what 1 and 2 refer to in these tests